src/houdini.h

```diff
@@ -25,8 +25,6 @@ extern "C" {
 
 extern bufsize_t houdini_unescape_ent(cmark_strbuf *ob, const uint8_t *src,
                                       bufsize_t size);
-extern int houdini_escape_html(cmark_strbuf *ob, const uint8_t *src,
-                               bufsize_t size);
```

Comment:
In addition to removing this, would it make sense to rename `houdini_escape_html0` to `houdini_escape_html`?

Reply:
Yes, this would make sense.

Reply:
I've done this.